app/src/main/res/values/strings.xml

```diff
@@ -105,22 +105,22 @@
     <string name="need_to_talk_message">I need to talk. Message sent through First Aide\'s Circle of Trust</string>
     <string name="msg_sent">Message sent</string>
     <string name="ok">OK</string>
-    <string name="message_failed">Message Sending Failed for Comrade </string>
+    <string name="message_failed">Message Sending Failed for Contact </string>
     <string name="no_phone_number">No phone number found</string>
     <string name="choose_number">Choose a number</string>
-    <string name="circle_of_trust_slide_two">When you are feeling insecure or need comrades\' attention.\nPress the button</string>
+    <string name="circle_of_trust_slide_two">When you are feeling insecure or need your emergenct contacts\' attention.\nPress the button</string>
     <string name="confirmation_message1">This message is being sent to</string>
-    <string name="confirmation_message2">registered Comrades.</string>
-    <string name="confirmation_message3">registered Comrade.</string>
-    <string name="no_comrade_title">No Comrades Registered</string>
-    <string name="no_comrade_msg">Failed to send message since no Comrades have been registered</string>
+    <string name="confirmation_message2">registered Contacts.</string>
+    <string name="confirmation_message3">registered Contact.</string>
+    <string name="no_comrade_title">No Contacts Registered</string>
+    <string name="no_comrade_msg">Failed to send message since no Contacts have been registered</string>
     <string name="add_comrades_save">Save</string>
     <string name="message_options">Message Template</string>
 
     <!--Trustees-->
-    <string name="updated_phone_numbers">"Comrades' phone numbers have been updated"</string>
+    <string name="updated_phone_numbers">"Your Emergency Contacts' phone numbers have been updated"</string>
```

Comment:
I am having second thoughts about that. Better to confirm with Sandi and Patrick. For now let's not capitalize that.  After you make the changes here, can you make a list of strings where you have doubts and let's confirm it with Sandi and Patrick.